### ISSUE TITLE
multipel users / multipel biometric keys

### ISSUE THREAD

**Issue by ralfgro** [Angelelz/WinHelloUnlock#70]
**Describe the bug**
This is less an issue as a question/request. We have a shared keepass DB on a network share, currently multiple users are unlocking the DB with a master password. Is it possible to use WinHelloUnlock with biometric data for each user or is only the data of the first user valid that is opening keepass after enabling the plugin? I didn't try it with multiple users, but I can't see any option to store data for multiple users.

**Comment by p-schneider** [comment]
If I understand the concept correctly it's not stored in your database at all. So multiple users should be able to use it. Just make sure the URI to the file is stable, so you're opening it from the same path/folder every time.

**Comment by Angelelz** [comment]
> If I understand the concept correctly it's not stored in your database at all. So multiple users should be able to use it. Just make sure the URI to the file is stable, so you're opening it from the same path/folder every time.

This is all correct! The data to unlock the database is stored with windows APIs not related to the database itself. And is stored locally, linked to the windows user.

**Comment by ralfgro** [comment]
Thx! Then I can safely enable this feature.
